content/en/docs/tutorials/hello-minikube.md

```diff
@@ -113,7 +113,7 @@ Pod runs a Container based on the provided Docker image.
 3. View the Pod:
 
     ```shell
-    kubectl get pods
+    kubectl get pods or kubectl get po -A
```

Comment:
```suggestion
    kubectl get pods -A
```
Trying to find a compromise that allows the example to still be copy/pasted without modification.